modules/data-entry/src/main/java/io/uhndata/cards/dataentry/internal/serialize/AbstractAnswerCopyProcessor.java

```diff
@@ -205,4 +141,12 @@ private Node findForm(final Node sourceForm, final Node question)
         }
         return null;
     }
+
+    abstract String getResourceName(Resource resource) throws RepositoryException;
+    abstract Node getAnswer(Node form, Node question);
+    abstract Node getSubject(Node source);
+    abstract Node getQuestionnaire(Node form);
+    abstract boolean isSubject(Node source);
+    abstract Node getOwnerQuestionnaire(Node question);
+    abstract Object getValue(Node answer);
```

Comment:
Minor code style change:

```suggestion
    abstract String getResourceName(Resource resource) throws RepositoryException;

    abstract Node getAnswer(Node form, Node question);

    abstract Node getSubject(Node source);

    abstract Node getQuestionnaire(Node form);

    abstract boolean isSubject(Node source);

    abstract Node getOwnerQuestionnaire(Node question);

    abstract Object getValue(Node answer);
```